config.js

```diff
@@ -258,7 +258,11 @@ config.iota = {
      * Default resource of the IoT Agent. This value must be different for every IoT Agent connecting to the IoT
      * Manager.
      */
-    defaultResource: '/iot/json'
+    defaultResource: '/iot/json',
+    /**
+     * flag indicating whether the incoming measures to the IoTAgent should be processed as per the "attributes" field.
+     */
+    defaultExplicitAttrs: true
```

Comment:
```suggestion
    explicitAttrs: true
```

as suggested in https://github.com/telefonicaid/iotagent-node-lib/pull/835/files#r363794493

Reply:
Fixed in 4724c32